xbmc/network/Socket.h

```diff
@@ -187,9 +187,9 @@ namespace SOCKETS
   {
   public:
     CPosixUDPSocket()
+    :  m_iSock(INVALID_SOCKET),
+       m_ipv6Socket()
```

Comment:
missing explicit value

Reply:
Works as is, because the bool will be zero inited, which means false. But this is really bad style. Should read `m_ipv6Socket(false)`